### DISCUSSION TITLE
Public directories in device group directories

### DISCUSSION THREAD

**Discussion by Avessalom608** [Ylianst/MeshCentral#5911] [Ideas]
I would like to suggest several ways to achieve the result:
1. Public directories are automatically created even in device group directories.

2. Give the opportunity to independently make the file publicly available, regardless of its location.

The point is that we have multiple administrators and they need to read/edit shared files (Via the administrative interface), but when
    1 administrator does not see the public files of another, this creates inconvenience

**Comment by si458** [comment]
You have already opened/asked this in the discussions question section
You need to open this is github issues and select feature request/enhancement
